### PR TITLE
PubMatic: Support hardcoded alias

### PR DESCRIPTION
**Pull request by pm-nilesh-chate**

### CODE REVIEW

File: adapters/pubmatic/pubmatic.go [hunk 1]
```diff
@@ -640,7 +641,8 @@ func getBidType(bidExt *pubmaticBidExt) openrtb_ext.BidType {
 // Builder builds a new instance of the Pubmatic adapter for the given bidder with the given config.
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
 	bidder := &PubmaticAdapter{
-		URI: config.Endpoint,
+		URI:        config.Endpoint,
+		bidderName: string(bidderName),
```
Comment (onkarvhanumante) [inline adapters/pubmatic/pubmatic.go:645]: @pm-nilesh-chate , can you add some description about how `bidderName` is planned to be used

**Comment by onkarvhanumante** [comment]
@pm-nilesh-chate  prebid server had major release 2.0

requesting to update `pm-nilesh-chate:master` with latest prebid master

File: adapters/pubmatic/pubmatic.go (hunk 1 above)
Comment (pm-nilesh-chate) [reply adapters/pubmatic/pubmatic.go:645]: This is for distinguishing between bidder and adapter name within the Fledge auction object, where hardcore aliases are used for PubMatic bidder which is part of PBS-Core. These changes related to PBS-Core appear straightforward. If you believe adding a comment would be beneficial, I'll include it in the next upcoming PR, which is also related to this.

**Comment by pm-nilesh-chate** [comment]
> @pm-nilesh-chate prebid server had major release 2.0
> 
> requesting to update `pm-nilesh-chate:master` with latest prebid master

done

**Comment by pm-nilesh-chate** [comment]
@onkarvhanumante  please review and merge this

**Review by onkarvhanumante — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**